docs/winui3.md

```diff
@@ -11,26 +11,29 @@ We are working on adding **WinUI 3** support for RNW and have made some good pro
 
 ## Current status
 
-As of June 2020, we support the **WinUI 3.0 Alpha** release for:
+As of June 25th 2020, we support the **WinUI 3.0 Preview 1 ** release for:

```

Comment:
Can you add a link to the WinUI 3 preview 1 nuget package (or) release notes (or) some place where folks can go to find out what it is and what's in it.